cpp/src/io/orc/stripe_data.cu

```diff
@@ -1108,7 +1108,7 @@ __global__ void __launch_bounds__(block_size)
   } temp_storage;
 
   orcdec_state_s* const s = &state_g;
-  const bool is_nulldec   = (blockIdx.y >= num_stripes);
+  bool const is_nulldec   = (blockIdx.y >= num_stripes);
   const uint32_t column   = blockIdx.x;
   const uint32_t stripe   = (is_nulldec) ? blockIdx.y - num_stripes : blockIdx.y;
   const uint32_t chunk_id = stripe * num_columns + column;
```

Comment:
I wonder why it skipped these

Reply:
Looks like, this is what clang-format warning was about.
There are other places where similar west const got replaced. So, this is a tool issue.